app/src/pages/_app.tsx

```diff
@@ -18,28 +20,21 @@ function MyApp({ Component, pageProps }: AppProps) {
     },
     eligibility: {
       residential: '',
-      pregnant: false,
-      baby: false,
-      child: false,
-      guardian: false,
-      none: false,
-      loss: false,
-      before: '',
-      insurance: false,
-      snap: false,
-      tanf: false,
-      fdpir: false,
-      none2: false,
+      categorical: [],
+      previouslyEnrolled: '',
+      adjunctive: [],
+    },
+    income: {
+      householdSize: '',
     },
   })
 
+  const props = { ...pageProps, session, setSession }
+
```

Comment:
@karinamzalez Since we only ever use `session` and `setSession` at the page level and never for nested component children, I've swapped us from using react contexts, to just passing these as top level props